# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/base/nsSyncLoadService.cpp
# Commit: 74f99033251c
# Full Hash: 74f99033251cd6c36eb6b0b5a4713eb6b5c793fa
# Author: Jonathan Kingston <jkt@mozilla.com>
# Date: 2019-02-13 09:48:26
# Regressor Bug: 1520868
# File Overlap Count: 1
# Description:
#   Bug 1520868 - Replacing AsyncOpen2 with AsyncOpen always r=valentin
#   
#   Replacing js and text occurences of asyncOpen2
#   Replacing open2 with open
#   
# ==============================================================================

diff -r 3378fd5f4f22 -r 74f99033251c dom/base/nsSyncLoadService.cpp
--- a/dom/base/nsSyncLoadService.cpp	Tue Feb 12 17:58:19 2019 +0100
+++ b/dom/base/nsSyncLoadService.cpp	Tue Feb 12 16:08:25 2019 +0000
@@ -205,7 +205,7 @@
   mAsyncLoadStatus = NS_OK;
 
   // Start reading from the channel
-  nsresult rv = mChannel->AsyncOpen2(this);
+  nsresult rv = mChannel->AsyncOpen(this);
 
   if (NS_SUCCEEDED(rv)) {
     // process events until we're finished.
@@ -230,7 +230,7 @@
 
 nsresult nsSyncLoader::PushSyncStream(nsIStreamListener *aListener) {
   nsCOMPtr<nsIInputStream> in;
-  nsresult rv = mChannel->Open2(getter_AddRefs(in));
+  nsresult rv = mChannel->Open(getter_AddRefs(in));
   NS_ENSURE_SUCCESS(rv, rv);
 
   mLoading = true;